pkg/services/libraryelements/libraryelements_create_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
-func TestCreateLibraryElement(t *testing.T) {
+func TestIntegration_CreateLibraryElement(t *testing.T) {
```

Comment:
Oh that's a good catch! We should go a hunt for all integration tests that pretend to be unit tests, but I haven't found an easy way to do it automatically (measure timing? integration tests usually take longer, but our unit tests can also be fairly creative in wasting CPU cycles)

Reply:
Hmm, interesting idea. What about something like this? https://github.com/grafana/grafana/pull/104463